Cargo.lock

```diff
@@ -2808,6 +2808,7 @@ version = "0.4.1"
 dependencies = [
  "ethers",
  "iron-types",
+ "log",
```

Comment:
you may need to merge `main` back here. this isnt' supposed to be here I think

Reply:
committed by mistake